packages/broker/src/helpers/scheduler.ts

```diff
@@ -0,0 +1,20 @@
+export const scheduleAtInterval = async (
```

Comment:
The scheduler does not do any error handling. The tasks methods **should** catch any errors internally.

As we currently don't have any global error handlers, throwing an error from `task()` will crash the application.

Reply:
Add this as a comment to the function description (i.e. JSdoc)

Reply:
Fixed